test/draw_test.py

```diff
@@ -6186,9 +6186,42 @@ def test_arc__invalid_color_formats(self):
             with self.assertRaises(TypeError):
                 bounds_rect = self.draw_arc(**kwargs)
 
-    def todo_test_arc(self):
+    def test_arc(self):
         """Ensure draw arc works correctly."""
-        self.fail()
```

Comment:
Instead, just remove:

```
def todo_test_arc(self):
     self.fail()
```

Reply:
Isn't that what they did?

Reply:
they also added a test here - but draw arc is already extensively tested.